### ISSUE TITLE
Install should include data

### ISSUE THREAD

**Issue by cnweaver** [arguelles/nuSQuIDS#21]
Currently, at configure time we fix the paths in global.h to point to the source directory, and do not change them when we install this header. As a result, a user can build and install the library, but then have it break down if the source directory (with the only copy of the default data tables) is subsequently deleted. We should probably:

1. Copy the data files to a suitable place like ${prefix}/share/nusquids
2. Update the paths in global.h when installing it to point to the installed data

**Comment by arguelles** [comment]
I think this is resolved in the dev branch, right @cnweaver ?

**Comment by cnweaver** [comment]
Correct, this is implemented on v1.11-dev; we may want to keep this open as a reminder until we merge that branch.
